<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_Change_Login_Screen</class>
 <widget class="QWidget" name="HNS_Change_Login_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="PB_ChangeLogin_ChUserPass">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>USER</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_ChangeLogin_PassReset">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>690</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>PASSWORD RESET</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_ChangeLogin_ChPass">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>510</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CHANGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_ChangeLogin_Reset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>650</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_ChangeLogin_Pass">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>430</y>
     <width>500</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141;border: 3px solid #414141;border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_ChangeLogin_Cancel">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>650</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_ChangeLogin_UserName">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>500</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141;border: 3px solid #414141;border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_ChangeLogin_ChUserName">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>160</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CHANGE</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_ChangeLogin_UserName">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>50</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>USERNAME</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_ChangeLogin_ChAdminPass">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>320</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>ADMIN</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_ChangeLogin_Save">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>650</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
